<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSender">
      <attribute name="title">
       <string>Sender</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>371</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Receiver ip:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditReceiverIp">
          <property name="text">
           <string>tcp://127.0.0.1:1666</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonConnect">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelConnectionStatus">
          <property name="text">
           <string>Connection status: not connected</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>371</width>
         <height>111</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLabel" name="labelFileToSend">
        <property name="text">
         <string>File to send:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonChooseFileToSend">
        <property name="text">
         <string>Choose file</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonSend">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBarSender">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tabReceiver">
      <attribute name="title">
       <string>Receiver</string>
      </attribute>
      <widget class="QSplitter" name="splitter_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>341</width>
         <height>111</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLabel" name="labelListeningStatus">
        <property name="text">
         <string>Listening status: not listening</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelReceiverIp">
        <property name="text">
         <string>IP: tcp://127.0.0.1:1666</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonStartListening">
        <property name="text">
         <string>Start listening</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelReceivinFile">
        <property name="text">
         <string>Receiving file:</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBarReceiver">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
